containers/SynonymList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { changeSynonyms } from '../actions'
+
+let SynonymList = ({ entityId, refId, synonyms, dispatch }) => (
+  <div>
+    <input 
+      type="text" 
+      placeholder="synonym list"
+      value={synonyms.join(";")}
```

Comment:
Uncaught TypeError: synonyms.join is not a function
I got this error while clicking on Add Ref